static/scss/partials/onboarding.scss

```diff
@@ -175,3 +174,235 @@
         }
     }
 }
+
+// Multi-part Premium Onboarding
+.is-premium-onboarding {
+
+    .c-multipart-premium-onboarding {
+        display: block;
+
+        @media #{$mq-md} {
+            display: flex;
+        }
+    }
+    
+    .dashboard-mpp-promo, .domain-alias-dashboard, .main-list, .mpp-dashbaord-header {
+        display: none;
+    }
+}
+
+.c-multipart-premium-onboarding {
+    display: none;
+    max-width: $content-lg;
+    margin-left: auto;
+    margin-right: auto;
+    position: relative;
+
+    @media #{$mq-md} {
+        min-height: calc(100vh - 150px);
+        flex-direction: column;
+        justify-content: space-between;
+    }
+
+    .c-button-as-text {
+        cursor: pointer;
+        appearance: none;
+        background-color: transparent;
+        border: none;
+        color: $color-informational;
+
+        @include text-body-sm;
+        margin-bottom: $spacing-sm;
+        
+        &.t-skip-button {
+            margin-bottom: 0;
+        }
+        
+
+        &:hover {
+            color: $color-informational-hover;
+        }
+    }
+
+    .c-premium-onboarding-content-image, .c-premium-onboarding-content-description {
+        @media #{$mq-md} {
+            width: 50%;
+            max-width: none;
+        }
+    }
+
+
+
+    .c-premium-onboarding-headlines {
+
+        h1 {
+            color: $color-purple-50;
+
+            @include text-title-sm;
+            margin-bottom: $spacing-md;
+        }
+
+        h2 {
+            @include text-title-3xs;
+        }
+
+        .c-premium-onboarding-content-description {
+            @include text-body-lg;
+        }
+    }
+
+    
+}
+
+
+
+.c-premium-onboarding-step {
+    display: none;
+
+    &.is-visible {
+        display: block;
+    }
+}
+
+// Step 2: Register custom subdomain 
+.c-premium-onboarding-step-2 {
+
+    samp {
+        @include text-title-2xs;
+        display: block;
+        color: $color-grey-30;
+        margin-bottom: $spacing-lg;
+
+        span {
+            color: $color-purple-50;
+            font-weight: 700;
+        }
+
+        @media #{$mq-md} {
+            @include text-title-3xs;
+        }
+        @media #{$mq-lg} {
+            @include text-title-2xs;
+        }
+    }
+}
+
+// Step 3: Install add-on
+firefox-private-relay-addon[data-addon-installed="true"] + firefox-private-relay-addon-data + main {
+
+    .c-premium-onboarding-install-addon {
+        display: none;
+    }
+
+    .c-premium-onboarding-addon-installed {
+        display: inline flex;
+    }
+
+    .c-premium-onboarding-action-step-3 {
+
+        .mzp-c-button {
+            display: inline-block;
+        }
+        
+        .c-button-as-text {
+            display: none;
+        }
+    }
+}
+
+.c-premium-onboarding-action-step-3 .mzp-c-button {
+    display: none;
+}
+
+.c-premium-onboarding-addon-installed {
+    padding: $spacing-md;
+    padding-inline-end: $spacing-lg;
```

Comment:
Wow, how is this the first time I've heard to this property? It's really neat!

Reply:
Yeah – Since we talked about start/end for stuff to be more RTL in the future.